From be8ef494586d9ef73c04ec4ca058a9a158ae3562 Mon Sep 17 00:00:00 2001
From: Sergei Trofimovich <slyich@gmail.com>
Date: Sun, 31 Dec 2023 22:26:31 +0000
Subject: [PATCH] src/node_type.h: add `#include <libxml/tree.h>` for
 `xmlNodePtr` type

Without the change build fails on `libxml2-2.12.3` as:

    ../../src/node_type.h:62:64: error: unknown type name 'xmlNodePtr'; did you mean 'nodePtr'?
       62 |         void            (*export)               (nodePtr node, xmlNodePtr cur, gboolean trusted);
          |                                                                ^~~~~~~~~~
          |                                                                nodePtr
---
 src/node_type.h | 1 +
 1 file changed, 1 insertion(+)

diff --git a/src/node_type.h b/src/node_type.h
index fcd3e7223..a41ff52fc 100644
--- a/src/node_type.h
+++ b/src/node_type.h
@@ -23,6 +23,7 @@
 
 #include "node.h"
 
+#include <libxml/tree.h>
 #include <libxml/xmlmemory.h>
 #include <gtk/gtk.h>
 
